docs/recipes/Internationalizing-Your-Module.md

```diff
@@ -2,6 +2,64 @@
 
 # Internationalizing Your Module
 
-Recipe is forthcoming.
+## Module Locale Files
+
+[one-app-locale-bundler](https://github.com/americanexpress/one-app-cli/tree/master/packages/one-app-locale-bundler) will bundle the locale files as a part of [one-app-bundler](https://github.com/americanexpress/one-app-cli/tree/master/packages/one-app-bundler). The [README](https://github.com/americanexpress/one-app-cli/blob/master/packages/one-app-locale-bundler/README.md) provides more information on the structure of locale files.
+
+## Utilizing Language Packs
+
+Each module must initialize it's own `IntlProvider`. [react-intl](https://github.com/formatjs/react-intl) version 3.x.x is compatible with One App and can be used to initialize an `IntlProvider` with a locale and language pack.
+
+To load a language pack leverage [loadLanguagePack](https://github.com/americanexpress/one-app-ducks) when initializing your module with [holocronModule](https://github.com/americanexpress/holocron).
```

Comment:
That's great you cited the method libraries. Could we also tighten it up to header links? (e.g. https://github.com/americanexpress/holocron/blob/master/packages/holocron/API.md#holocronmodule)

Reply:
Sure